coreos-installer

```diff
@@ -904,6 +904,11 @@ main() {
     # by a few of the functions below.
     DEST_DEV=$(cat /tmp/selected_dev)
     DEST_DEV=/dev/$DEST_DEV
+    REAL_DEV=`readlink -n -f $DEST_DEV || echo -n $DEST_DEV`
```

Comment:
How about:
```
if [ -L "${DEST_DEV}" ]; then
  DEST_DEV=$(realpath $DEST_DEV)
fi
```
?

Reply:
Thx, good idea. The reason why i did it a little bit different - to `echo` a log message, which could be useful for user.